packages/components/src/Layout/SkipLinks/SkipLinks.component.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'react-i18next';
+
+import Icon from '../../Icon';
+import I18N_DOMAIN_COMPONENTS from '../../constants';
+import getDefaultT from '../../translate';
+
+import theme from './SkipLinks.scss';
+
+function SkipTo({ href, label }) {
+	return (
+		<a href={href}>
+			<span className={theme.icon}>
+				<Icon transform="rotate-270" name="talend-arrow-left" />
+			</span>
+			<span className={theme.text}>{label}</span>
+		</a>
+	);
+}
+SkipTo.propTypes = {
+	href: PropTypes.string.isRequired,
+	label: PropTypes.string.isRequired,
+};
+
+/**
+ * Skip links are a technique to ease keyboard navigation.
+ * It consists in giving some links as the firsts focusable elements of the page
+ * to go directly to some meaningful content.
+ */
+function SkipLinks({ mainId, navigationId, t }) {
+	return (
+		<nav
+			className={theme['tc-skip-links']}
+			aria-label={t('SKIP_LINKS', { defaultValue: 'Skip links' })}
+		>
+			<ul>
+				{navigationId && (
```

Comment:
There's a defaultProp here so we don't need this conditional. And I don't think we need to make it optional unless we specifically need it disabled in an app.

Reply:
as we can disable them we need them. For example, a page where we have a layout with only a main, and no navigation side panel

Reply:
In fact, since we use it explicitly in Layout component, I'll remove the default props, so the api is better.
By default we don't want it, and if the id is explicitly passed, we add the link